docs/providers/aws/guide/serverless.yml.md

```diff
@@ -44,9 +44,10 @@ provider:
   logRetentionInDays: 14 # Set the default RetentionInDays for a CloudWatch LogGroup
   kmsKeyArn: arn:aws:kms:us-east-1:XXXXXX:key/some-hash # KMS key arn which will be used for encryption for all functions
   deploymentBucket:
+    blockPublicAccess: true # Prevents public access via ACLs or bucket policies. Default is false
+    disableDefaultBucketPolicyCreation: false # Prevents creation of default bucket policy when framework creates the deployment bucket. Default is false
```

Comment:
Sorry for second thought, I've just looked at this property name and thought that probably we could improve it to `skipPolicySetup` (?)

Note that we're already in context of `deploymentBucket` configuration, so _"bucket"_ part seems superfluous, and it's more about skipping some part instead of disabling it

Reply:
I was honestly not sure about the setting name and I was a bit worried it's too long and verbose - I like your proposal, will change it :100: